specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```diff
@@ -5642,6 +5753,59 @@
       },
       "description": "An object that defines the blob inventory rule filter conditions. For 'Blob' definition.objectType all filter properties are applicable, 'blobTypes' is required and others are optional. For 'Container' definition.objectType only prefixMatch is applicable and is optional."
     },
+    "StorageAccountMigration": {
+      "properties": {
+        "id": {
```

Comment:
from example, it looks "id" won't be input from request, but will be output from server. So it might should be "readOnly": true

Reply:
added readOnly flag